core/src/main/resources/error/error-classes.json

```
@@ -1580,6 +1580,11 @@
       "Not allowed to implement multiple UDF interfaces, UDF class <className>."
     ]
   },
+  "NAMED_ARGUMENTS_SUPPORT_DISABLED" : {
+    "message" : [
+      "Cannot call function ‘<functionName>’ because named argument references are not supported here. In this case, the named argument reference was ‘<argument>’.\n"
```

```suggestion
      "Cannot call function <functionName> because named argument references are not supported here. In this case, the named argument reference was <argument>"
```

We should avoid quoting directly in the error message. Instead, in the `sql/catalyst/src/main/scala/org/apache/spark/sql/errors/QueryCompilationErrors.scala`, try to wrap those idenfitiers using `toSQLId`.